src/generated/todo/index.js

```diff
@@ -16,8 +16,8 @@ const actions = Object.assign(
 );
 
 const sagas = [].concat(
```

Comment:
Could also have each component's sagas export an object instead of an array, and have `allSagas` merge sagas across components.

```javascript
const sagas = {
  getTodosEverySaga: getTodosSaga.takeEverySaga,
  postTodoLatestSaga: postTodoSaga.takeLatestSaga,
}
export {
  actionTypes,
  actions,
  sagas,
}
```
Still unsure of where the logic to choose a component's saga (between `takeEvery` and `takeLatest`) will sit.